cliff.toml

```diff
@@ -0,0 +1,65 @@
+# configuration file for git-cliff (0.1.0)
+
+[changelog]
+# changelog header
+header = """
+# Changelog\n
+All notable changes to this project will be documented in this file.\n
+"""
+# template for the changelog body
+# https://tera.netlify.app/docs/#introduction
+body = """
+{% if version %}\
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
+    ## [unreleased]
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | upper_first }}
+    {% for commit in commits %}
+        - {% if commit.breaking %}[**breaking**] {% endif %}{{ commit.message | upper_first }}\
```

Comment:
```suggestion
        - {% if commit.breaking %}[**BREAKING**] {% endif %}{{ commit.message | upper_first }}\
```

Reply:
Doesn't hurt to have a bit more prominence?

Reply:
I don't think I want the changelog to be this shouty.